.github/workflows/secrets-detect.yml

```diff
@@ -0,0 +1,17 @@
+
+name: Secrets detection using detect-secrets
+on:
+  pull_request:
+    branches: [ "master" ]
```

Comment:
I don't think we actually use the `master` branch of `jellyfish`? Should this be `develop`?

Reply:
Agreed! But this is `jf_agent` where we don't use develop. :)